tests/baselines/reference/user/lodash.log

```diff
@@ -5,28 +5,40 @@ node_modules/lodash/_ListCache.js(20,17): error TS2532: Object is possibly 'unde
 node_modules/lodash/_MapCache.js(20,17): error TS2532: Object is possibly 'undefined'.
 node_modules/lodash/_SetCache.js(19,14): error TS2532: Object is possibly 'undefined'.
 node_modules/lodash/_Stack.js(17,20): error TS2339: Property 'size' does not exist on type 'ListCache'.
+node_modules/lodash/_arrayAggregator.js(11,33): error TS1016: A required parameter cannot follow an optional parameter.
```

Comment:
lodash consistently uses brackets to mean that the type includes `null | undefined`, *not* that it's possibly missing. eg

```js
/** @param {Array} [array] The array to iterate over */
```

would be written in TS as

```ts
array: Array<unknown> | null | undefined
```